sdk/objectanchors/Azure.MixedReality.ObjectAnchors/README.md

```diff
@@ -0,0 +1,144 @@
+    - [Begin Object Anchors asset conversion after uploading an asset](#begin-object-anchors-asset-conversion-after-uploading-an-asset)
+    - [Poll an existing ObjectAnchors asset conversion until completion and download the result](#poll-an-existing-objectanchors-asset-conversion-until-completion-and-download-the-result)
+  - [Troubleshooting](#troubleshooting)
+  - [Next steps](#next-steps)
+  - [Contributing](#contributing)
+
+## Getting started
+
+### Install the package
+
+Install the Azure Object Anchors client library for .NET using one of the following methods.
+
+From Visual Studio Package Manager:
+
+```powershell
+Install-Package Azure.MixedReality.ObjectAnchors
+```
+
+From .NET CLI
+
+```dotnetcli
+dotnet add package Azure.MixedReality.ObjectAnchors
+```
+
+Add a package reference:
+
+```xml
+<PackageReference Include="Azure.MixedReality.ObjectAnchors" Version="0.1.0-beta.0" />
+```
+
+### Prerequisites
+
+- You must have an [Azure subscription](https://azure.microsoft.com/free/).
+- You must have an [Azure Object Anchors](https://review.docs.microsoft.com/azure/object-anchors/) account.
+
+### Authenticate the client
+
+Mixed Reality services support a few different forms of authentication:
+
+- Account Key authentication
+  - Account keys enable you to get started quickly with using Mixed Reality services. But before you deploy your application
+    to production, we recommend that you update your app to use Azure AD authentication.
+- Azure Active Directory (AD) token authentication
+  - If you're building an enterprise application and your company is using Azure AD as its identity system, you can use
+    user-based Azure AD authentication in your app. You then grant access to your Object Anchors account by using your
+    existing Azure AD security groups. You can also grant access directly to users in your organization.
+  - Otherwise, we recommend that you obtain Azure AD tokens from a web service that supports your app. We recommend this
+    method for production applications because it allows you to avoid embedding the credentials for access to the Object Anchors asset conversion service in your client application.
+
+See [here](https://docs.microsoft.com/azure/spatial-anchors/concepts/authentication) for detailed instructions and information.
+
+## Key concepts
+
+### ObjectAnchorsClient
+
+The `ObjectAnchorsClient` is the client library used to access the Object Anchors asset conversion service. From there, a storage upload URI will be provided for users to upload their assets for conversion into a format that is usable by the Object Anchors runtime.
+
+Assets uploaded to the Microsoft-hosted URI obtained from the client will be retained for **48 hours**.
+
+The final converted model in Microsoft-hosted storage will be retained for **48 hours**. 
+
+## Examples
+
+### Upload an asset for Object Anchors asset conversion
+
+```csharp
+TokenCredential credential = new DefaultAzureCredential(true);
```

Comment:
>DefaultAzureCredential [](start = 33, length = 22)

I'd prefer we instead document this using the `AzureKeyCredential`. It's the easiest way for people to quickly get started and what we'll be using in the quickstarts.

Reply:
Pending

Reply:
Replaced with `AzureKeyCredential`